src/error.js

```diff
@@ -54,18 +54,35 @@ let globalExponentialBackoff = function(work) {
  * elements instead of stringification.
  * @param {*} error
  * @param {!Element=} opt_associatedElement
+ * @return {!Error}
  */
 export function reportError(error, opt_associatedElement) {
-  if (!self.console) {
-    return;
+  // Convert error to the expected type.
+  let isValidError;
+  if (error) {
+    if (typeof error == 'string') {
+      error = new Error(error);
+    } else {
+      isValidError = true;
+    }
+  } else {
+    error = new Error('Unknown error');
   }
-  if (!error) {
-    error = new Error('no error supplied');
+  // Report if error is not an expected type.
+  if (!isValidError && getMode().localDev) {
+    setTimeout(function() {
+      const rethrow = new Error('Error reported incorrectly: ' + error);
+      rethrow.reported = true;
```

Comment:
Hmm. I do see it in console right now. But I changed it to do `_reported_` - that one is only ignored for sending to server.